metadata-models/src/main/pegasus/com/linkedin/common/GlossaryTermAssociation.pdl

```diff
@@ -7,5 +7,10 @@ record GlossaryTermAssociation {
    /**
     * Urn of the applied glossary term
     */
+    @Searchable = {
+        "fieldName": "glossaryTerms",
+        "fieldType": "TEXT_PARTIAL",
```

Comment:
lets make the fieldType URN_PARTIAL- it parses urns more correctly than TEXT_PARTIAL does

Reply:
Yes i had considered using URN_PARTIAL but our requirement was something different
Lets say we have a urn -> urn:li:glossaryTerm:example.balanceAmount_v1 where exmaple is a domain
In this case URN_PARTIAL will search whatever start with examp......
But we dont want to consider domain while searching, and want that user can search using just balanceAmount_v1 which cant be achieved by URN_PARTIAL.
We are looking forward to create two different key **domain and name**  instead of hardcoding domain in name itself
Once we will come with that implementation we can replace TEXT_PARTIAL with URN_PARTIAL
Till then can we keep TEXT_PARTIAL as fieldType ?

Reply:
> @lalrishav it looks like a comment you added may have been deleted- are you still planning on using TEXT_PARTIAL instead of TEXT_URN as a temporary fix for partial term searches?

yes can we do that pls